### DISCUSSION TITLE
v-datatable, move/mirror horizontal scroll to the top

### DISCUSSION THREAD

**Discussion by lasteuropean** [vuetifyjs/vuetify#12483] [Q&A]
![datatable](https://user-images.githubusercontent.com/9833790/96911333-f5021300-145d-11eb-98f9-1295eadece65.png)

Attached picture shows what I'm going for. Ideally I'd have two scroll bars;one at the top _and_ bottom, but I'd settle for moving it to the top. Project is in Vue/Nuxt, so I'd prefer to avoid JQuery. Wondering if Vuetify has a solution I'm missing.

**Comment by sugoidesune** [comment]
You will have to replace the browser based scrollbar with a javascript scrollbar.
This vue library will help:
https://binaryify.github.io/vue-custom-scrollbar/en/

**Reply by darkbasic** [reply]
@sugoidesune could you please make an example?

**Reply by sugoidesune** [reply]
Sorry no longer have access to a vuetify project and I'm no longer sure if this is the best solution.
